web/annotation_editor_layer_builder.css

```
@@ -529,13 +529,54 @@
     mask-image: var(--alt-text-done-image);
   }
 
-  & .description {
-    position: absolute;
-    top: 0;
-    left: 0;
+  & .tooltip {
     display: none;
-    width: 0;
-    height: 0;
+  }
+
+  & .tooltip.show {
+    --alt-text-tooltip-bg: #f0f0f4;
+    --alt-text-tooltip-fg: #15141a;
+    --alt-text-tooltip-border: #8f8f9d;
+    --alt-text-tooltip-shadow: 0px 2px 6px 0px rgba(58, 57, 68, 0.2);
+
+    @media (prefers-color-scheme: dark) {
+      :root {
+        --alt-text-tooltip-bg: #1c1b22;
+        --alt-text-tooltip-fg: #fbfbfe;
+        --alt-text-tooltip-border: #8f8f9d;
+        --alt-text-tooltip-shadow: 0px 2px 6px 0px #15141a;
+      }
+    }
+
+    @media screen and (forced-colors: active) {
+      :root {
+        --alt-text-tooltip-bg: Canvas;
+        --alt-text-tooltip-fg: CanvasText;
+        --alt-text-tooltip-border: CanvasText;
+        --alt-text-tooltip-shadow: none;
+      }
```

Similarly, shouldn't this also be changed as follows?
```suggestion
      --alt-text-tooltip-bg: Canvas;
      --alt-text-tooltip-fg: CanvasText;
      --alt-text-tooltip-border: CanvasText;
      --alt-text-tooltip-shadow: none;
```